scss/core/base.scss

```diff
@@ -352,7 +352,7 @@ table.mce-item-table {
 
 // Outline
 
-.focus-outline:focus, .focus-outline:focus-within {
+html.no-touchevents .focus-outline:focus, html.no-touchevents .focus-outline:focus-within {
   outline: none;
```

Comment:
Why does `outline: none` need to be repeated so many times here?

Reply:
Fixed